src/main/twirl/gitbucket/core/repo/upload.scala.html

```diff
@@ -56,7 +56,7 @@
     clickable: true,
     previewTemplate: "<div class=\"dz-preview\">\n  <div class=\"dz-progress\"><span class=\"dz-upload\" data-dz-uploadprogress>Uploading your files...</span></div>\n  <div class=\"dz-error-message\"><span data-dz-errormessage></span></div>\n</div>",
     success: function(file, id) {
-      file.previewElement.remove();
+      $(file.previewElement).remove();
```

Comment:
For SCRIPT438 error
![image](https://user-images.githubusercontent.com/17608272/82119299-32c88480-97b8-11ea-973b-36791a9d9dd3.png)